### PR TITLE
rbac: remove dependency on statics and config packages

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: config/rbac.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
```
Comment (safchain) [inline config/rbac.go]: copyright doesn't seems good

**Comment by safchain** [comment]
run vagrant-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run vagrant-tests

**Pull request merged**